frontend/test/playwright/playwright.config.ts

```diff
@@ -26,6 +26,7 @@ const pwCommand =
   process.env.FASTSTART !== "false" ? "start:playwright" : "prod:playwright"
 
 const config: PlaywrightTestConfig = {
+  forbidOnly: !!process.env.CI,
```

Comment:
This allows it to pass locally but fails CI if `.only` is present.

Reply:
Why would we allow it to pass locally if `.only` is present but fail on CI? Is it time consuming to have this check run locally? And do you think it makes sense to add a comment as to why this was added here?